internal/btf/types.go

```diff
@@ -233,15 +233,22 @@ func (f *Func) copy() Type {
 type FuncProto struct {
 	TypeID
 	Return Type
-	// Parameters not supported yet
+	Params []FuncParam
 }
 
 func (fp *FuncProto) walk(cs *copyStack) { cs.push(&fp.Return) }
```

Comment:
I think this needs to push all fp.Params.Type instances as well.

Reply:
Updated.